consoles/virtio_screen.pm

```diff
@@ -197,7 +199,7 @@ sub read_until {
     bmwqemu::log_call(%nargs);
 
     my $rin = '';
-    vec($rin, fileno($fd), 1) = 1;
+    vec($rin, fileno($sel_fd), 1) = 1;
```

Comment:
Or we just merge https://github.com/os-autoinst/os-autoinst/pull/1012 first. I would know how to resolve the conflict when rebasing.

Reply:
My concern is that you may have to revert the virtio_screen/console changes which is more complicated if the SSH console is also merged. However it is probably quite a low risk.

Reply:
In fact the changes I made here are contained by https://github.com/os-autoinst/os-autoinst/pull/1012. Resolving the conflicts would simply mean to discard all of my changes in `virtio_screen.pm` and use `consoles::serial_screen` instead of `consoles::virtio_screen` in `sshVirtshSUT.pm`.